### ISSUE TITLE
TypeError when logging exceptions

### ISSUE THREAD

**Issue by kristaps** [getsentry/raven-python#627]
Some change introduced in 5.4.3 causes TypeErrors in our app every time an exception is logged.

This is a typical traceback with app code until logger invocation removed:
```
 File "[our_app].py", line 1126, in [some_function]
    logger.exception("Foo")
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 1193, in exception
    self.error(msg, *args, **kwargs)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 1186, in error
    self._log(ERROR, msg, args, **kwargs)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 1279, in _log
    self.handle(record)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 1289, in handle
    self.callHandlers(record)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 1329, in callHandlers
    hdlr.handle(record)
  File "/usr/local/Cellar/python/2.7.9/Frameworks/Python.framework/Versions/2.7/lib/python2.7/logging/__init__.py", line 757, in handle
    self.emit(record)
  File "/Users/me/.virtualenvs/our_app/lib/python2.7/site-packages/raven/handlers/logging.py", line 70, in emit
    if self.client.raise_send_errors:
  File "/Users/me/.virtualenvs/our_app/lib/python2.7/site-packages/raven/contrib/django/models.py", line 54, in <lambda>
    __getattr__ = lambda x, o: getattr(get_client(), o)
  File "/Users/me/.virtualenvs/our_app/lib/python2.7/site-packages/raven/contrib/django/models.py", line 150, in get_client
    Client = import_string(client)
  File "/Users/me/.virtualenvs/our_app/lib/python2.7/site-packages/raven/utils/imports.py", line 9, in import_string
    module = __import__(module_name, {}, {}, [class_name])
TypeError: Item in ``from list'' not a string
```

The relevant settings.LOGGING['handlers'] snippet:
```
        'sentry': {
            'level': 'ERROR',
            'filters': ['require_debug_false'],
            'class': 'raven.contrib.django.raven_compat.handlers.SentryHandler',
        },
```

Please let me know if you need any other details.

**Comment by dcramer** [comment]
Are you using unicode literals future import? This would suggest the 'class' here is a unicode string, and we dont coerce it to a bytestring, which is required for imports.

**Comment by kristaps** [comment]
I found exactly one unicode_literals instance in the application code, but removing it doesn't fix the TypeErrors.

Besides that, all of our migrations which are generated by Django use unicode_literals import, but that code should not be imported during normal operation.

**Comment by dcramer** [comment]
I'm not entirely sure why something wouldn't be a string here, but we'll put in a bit to force statements to a string in Python 2.

**Comment by chriskief** [comment]
Also seeing this issue. Not using unicode literals.

**Comment by dcramer** [comment]
I've pushed the given fix to 5.4.4.
